ros/src/simulation/gazebo_simulator/launcher/vehicle_gazebo_simulation_launcher/scripts/download_model.sh

```diff
@@ -1,25 +1,22 @@
 #!/bin/sh
 
 mkdir -p ~/.gazebo/models
-echo "Download generic gazebo models"
+echo "---- Download generic gazebo models ----"
 if [ -e /tmp/gazebo_models ]; then
-  echo "Already exist /tmp/gazebo_models"
-  echo "removing /tmp/gazebo_models"
+  echo "---- Already exist /tmp/gazebo_models ----"
+  echo "---- removing /tmp/gazebo_models ----"
   rm -r /tmp/gazebo_models
 fi
 hg clone https://bitbucket.org/osrf/gazebo_models /tmp/gazebo_models
 
 if [ -e /tmp/gazebo_models ]; then
-  echo "Install generic gazebo models"
+  echo "---- Install generic gazebo models ----"
   mv -n /tmp/gazebo_models/* ~/.gazebo/models/
-  echo "Completed"
+  echo "---- Completed ----"
 else
-  echo "Error"
+  echo "---- Error ----"
 fi
 
-echo "Install osrf_citysim models"
-SCRIPT_DIR=$(cd $(dirname $0); pwd)
-cp -rn $SCRIPT_DIR/../../../worlds/external/osrf_citysim/models/* ~/.gazebo/models/
-
-echo "Install car_demo models"
-cp -rn $SCRIPT_DIR/../../../worlds/external/car_demo/car_demo/models/* ~/.gazebo/models/
+echo "---- Install models ----"
+cp -rn $(rospack find gazebo_world_description)/models/* ~/.gazebo/models/
```

Comment:
why are you forcing the download? this is wrong in my opinion. what do you think @sgermanserrano ?
this will require to download extra whopping ~450 MB for every installation. (https://bitbucket.org/osrf/gazebo_models/downloads/)
This script should be optional, only users who wish to use the simulator should be executing this script.

Reply:
Thank you @amc-nu . I talked with him, but it looks like it's better to change to symlink